src/lib.rs

```diff
@@ -174,13 +174,29 @@ where
         }
     }
 
-    /// Subscribes to the refreshable.
+    /// Subscribes to the refreshable with an infallible callback.
+    ///
+    /// The callback will be invoked every time the refreshable's value changes, and is also called synchronously when
+    /// this method is called with the current value.
+    pub fn subscribe<F>(&self, mut callback: F) -> Subscription<T, E>
+    where
+        F: FnMut(&T) + 'static + Sync + Send,
+    {
+        self.try_subscribe(move |value| {
+            callback(value);
+            Ok(())
+        })
+        .ok()
+        .unwrap()
+    }
+
+    /// Subscribes to the refreshable with a fallible callback.
     ///
     /// The callback will be invoked every time the refreshable's value changes, and is also called synchronously when
     /// this method is called with the current value. If the callback returns `Ok`, a `Subscription` object is returned
     /// that will unsubscribe from the refreshable when it drops. If the callback returns `Err`, this method will return
     /// the error and the callback will *not* be invoked on updates to the value.
```

Comment:
These are existing docs, but I think that it should probably be specified what happens if an error is returned on an invocation of the callback that is not the first invocation?

Reply:
Yep, will clarify.

Reply:
Updated